README.md

```diff
@@ -4,6 +4,53 @@ libsignal-client is an implementation of the Signal client in Rust.
 
 Work in progress.  Subject to change without notice, use outside Signal not yet recommended.
 
+# Building
+
+To build anything in this repository you must have Rust installed. The build currently
+uses a specific version of the Rust nightly compiler (though probably most recent Rust
+nightlies are fine).
+
+```$ rustup toolchain add nightly-2020-11-09```
```

Comment:
Looks like you don't even have to do this; `cargo build` starts downloading the toolchain in `rust-toolchain` if you don't have it installed.